hack/update-generated-protobuf.sh

```diff
@@ -37,7 +37,13 @@ function prereqs() {
   KUBE_BUILD_CONTAINER_NAME="kube-build-${KUBE_ROOT_HASH}"
   KUBE_BUILD_DATA_CONTAINER_NAME="kube-build-data-${KUBE_ROOT_HASH}"
   DOCKER_MOUNT_ARGS=(
-    --volume "${REPO_DIR:-${KUBE_ROOT}}:/go/src/${KUBE_GO_PACKAGE}"
+    --volume "${REPO_DIR:-${KUBE_ROOT}}/cluster:/go/src/${KUBE_GO_PACKAGE}/cluster"
```

Comment:
1) PLEASE add a comment explaining why the *obvious* cleanup doesn't work.

2) Please explain to me why it doesn't work?  From the error you pasted before it looks like a root/non-root issue.  I didn't have any such issue here - did someone just need to run 'make clean' or 'sudo make clean'.  Nothing in our build should actually require root.